apps/framework-cli/src/framework/python/templates.rs

```diff
@@ -88,7 +88,7 @@ def run(client, params):
     )
 "#;
 
-pub static PTYHON_BASE_BLOCKS_TEMPLATE: &str = r#"
+pub static PYTHON_BASE_BLOCKS_TEMPLATE: &str = r#"
 from dataclasses import dataclass
 from typing import List
```

Comment:
What about this? does this need to be updated?

Reply:
it's a typo fix :P